README.md

```diff
@@ -129,9 +129,9 @@ Check out our FAQs and get answers to common questions:
 
 + https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions
 
-Talk to us on IRC:
+Talk to us on IRC(Internet Relay Chat):
```

Comment:
There's a space missing between `IRC` and `(Internet Relay Chat)`.

Reply:
edited as requested